src/hearings/containers/request-hearing/hearing-panel/hearing-panel.component.ts

```diff
@@ -201,10 +200,13 @@ export class HearingPanelComponent extends RequestHearingPageFlow implements OnI
         key: control.value.key,
         value_en: control.value.value_en,
         value_cy: control.value.value_cy,
-        hintText_EN: control.value.hintText_EN,
-        hintTextCY: control.value.hintTextCY,
-        order: control.value.order,
-        parentKey: control.value.parentKey,
+        hint_text_en: control.value.hint_text_en,
```

Comment:
is it worth trying to use {...control.value} instead of each key assignment as the source and destination object holding the same keys

Reply:
thanks that is a good point, absolutely we can do that, however I will keep the existing code for data transformation to avoid retest the whole journey.